### PR TITLE
Add source map settings to the ADO symbol server docs

### PR DESCRIPTION
**Pull request by captainbrosset**
This is based on #2225 by @Meir017. The only changes I've made are a slightly rephrased bullet point, and a new image that looks more like the other image in the same section.

Rendered page for review: https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure?branch=pr-en-us-2227

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md [hunk 1]
```diff
@@ -74,7 +74,13 @@ To configure DevTools:
 
 1. To open DevTools, in Microsoft Edge, right-click a webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).
 
-1. In DevTools, click **Settings** (![Settings icon.](../media/settings-gear-icon-light-theme.png)) > **Symbol Server**.
+1. In DevTools, click **Settings** (![Settings icon.](../media/settings-gear-icon-light-theme.png)) > **Preferences**.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md:77]:
```suggestion
1. In DevTools, click **Settings** (![Settings icon](../media/settings-gear-icon-light-theme.png)) > **Preferences**.
```

File: microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md [hunk 2]
```diff
@@ -74,7 +74,13 @@ To configure DevTools:
...
+
+1. Make sure the **Enable JavaScript source maps** checkbox and the **Enable CSS source maps** checkbox are selected.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md:79]:
```suggestion
1. Make sure the **Enable JavaScript source maps** checkbox and the **Enable CSS source maps** checkbox are selected:
```

File: microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md [hunk 3]
```diff
@@ -74,7 +74,13 @@ To configure DevTools:
...
+
+    ![The Preferences configuration screen in DevTools settings](images/ado-preferences-source-maps-devtools.png)
+
+1. In the sidebar of the **Settings** page, click **Symbol Server**.
 
 1. In the **Azure DevOps organization** text box, enter the Azure DevOps organization where you created the PAT.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md:85]:
```suggestion
1. In the **Azure DevOps organization** text box, enter the Azure DevOps organization where you created the personal access token (PAT).
```

**Review by Meir017 — APPROVED** [review]

**Review by JasonAndrewWriter — APPROVED** [review]

**Comment by captainbrosset** [comment]
@Meir017 thank you for getting this started, this is now merged in and should be live in production momentarily!

**Pull request merged**